server/mediaserver.go

```diff
@@ -578,6 +585,11 @@ func getRTMPStreamHandler(s *LivepeerServer) func(url *url.URL) (stream.RTMPVide
 
 // HandlePush processes request for HTTP ingest
 func (s *LivepeerServer) HandlePush(w http.ResponseWriter, r *http.Request) {
+	if !s.HTTPIngest {
+		respondWith400(w, "HTTP ingest is disabled")
```

Comment:
Arguably an HTTP 405 "Method Not Allowed" but idk if we observe that sort of thing at all in go-livepeer

Reply:
Fixed in 727d98b